libs/commands/src/services/command-handler.service.spec.ts

```diff
@@ -35,7 +40,15 @@ describe('CommandsService', () => {
         expect(service).toBeDefined();
     });
 
-    describe('CommandHandler.execute', () => {
+    describe('CommandHandlerService.getAllCommandCtorsAndMetadata', () => {
+        it('should return the expected meta', () => {
+            const result = service.getAllCommandCtorsAndMetadata();
+
+            expect(result).toMatchSnapshot();
```

Comment:
Check the ctor is correct because the snapshot doesn't seem to be specific enough.

Reply:
done